src/ORM/DataObject.php

```diff
@@ -3175,14 +3175,24 @@ public static function reset()
     /**
      * Return the given element, searching by ID
      *
-     * @param string $callerClass The class of the object to be returned
+     * @param string $callerClass The class of the object to be returned (optional if called from extending class)
      * @param int $id The id of the element
      * @param boolean $cache See {@link get_one()}
```

Comment:
Clarify if calling with `MyClass::get_by_id($id)`. Does this become second arg, or do you pass null for second arg and this stays third?